src/domain/notifications/v2/entities/upsert-subscriptions.dto.entity.ts

```diff
@@ -10,5 +10,5 @@ export type UpsertSubscriptionsDto = {
     notificationTypes: Array<NotificationType>;
   }>;
   deviceType: DeviceType;
-  deviceUuid?: Uuid;
+  deviceUuid?: UUID;
```

Comment:
If we add (and use) a schema for this DTO, using the `UuidSchema` here, we won't need to cast in places.